src/index.ts

```diff
@@ -1,7 +1,7 @@
 import {startServer} from './server';
 import {extract} from './text';
```

Comment:
```suggestion
export {startServer} from './server';
export {extract} from './text';
```

Reply:
that's actually was my initial idea, but I thought it's no good since it's changing the way it needs to be imported further